api.py

```diff
@@ -22,9 +22,13 @@ async def status():
 
 logger = get_logger()
 
-AJV_HOST = os.getenv("AJV_HOST", "localhost")
+AJV_VALIDATOR_SCHEME = os.getenv("AJV_VALIDATOR_SCHEME", "http")
+AJV_VALIDATOR_HOST = os.getenv("AJV_VALIDATOR_HOST", "localhost")
+AJV_VALIDATOR_PORT = os.getenv("AJV_VALIDATOR_PORT", "5002")
 
-AJV_VALIDATOR_URL = f"http://{AJV_HOST}:5002/validate"
+AJV_VALIDATOR_URL = (
+    f"{AJV_VALIDATOR_SCHEME}://{AJV_VALIDATOR_HOST}:{AJV_VALIDATOR_PORT}/validate"
```

Comment:
Can we make it -
f"{AJV_VALIDATOR_SCHEME}://{AJV_VALIDATOR_HOST}/validate"
When deploying in GCP we won't be using port at all as the load balancer will redirect to the service.